packages/web-component/src/lib/descope-wc/BaseDescopeWc.ts

```diff
@@ -155,6 +154,11 @@ class BaseDescopeWc extends HTMLElement {
     return res === 'true';
   }
 
+  get preferBiometrics(): boolean {
+    const res = this.getAttribute('prefer-biometrics') ?? 'true';
```

Comment:
we should probably add it to readme with all other properties 
![image](https://github.com/descope/descope-js/assets/10514677/4c0b9279-b628-474a-b9d1-58e58fddf234)